nf_core/pipelines/launch.py

```diff
@@ -406,10 +406,21 @@ def prompt_schema(self):
         """Go through the pipeline schema and prompt user to change defaults"""
         answers = {}
         # Start with the subschema in the definitions - use order of allOf
-        definitions_schemas = self.schema_obj.schema.get("$defs", self.schema_obj.schema.get("definitions", {})).items()
+        defs_notation = self.schema_obj.defs_notation
+        log.debug(f"defs_notation: {defs_notation}")
+        definitions_schemas = self.schema_obj.schema.get(defs_notation, {}).items()
```

Comment:
do we need to use `items()` here? I think we can get the dictionary directly
```suggestion
        definitions_schemas = self.schema_obj.schema.get(defs_notation, {})
```

Reply:
good catch!